instance/models/mixins/domain_names.py

```diff
@@ -256,7 +264,7 @@ def lms_extended_heartbeat_url(self):
         """
         return u'{}heartbeat?extended'.format(self.url)
 
-    def save(self, **kwargs):  # pylint: disable=arguments-differ, too-many-branches, useless-suppression
+    def save(self, **kwargs):  # pylint: disable=arguments-differ, too-many-branches, useless-suppression; # noqa: MC0001
```

Comment:
Just a drive-by comment, but I think we should prefer using a human-readable name here instead of MC0001

Reply:
@xirdneh I'm unable to find the human-readable version of `MC0001` in the [repo](https://github.com/PyCQA/mccabe).

Reply:
@michaelbukachi I think you pinged the wrong person there :-) 

Maybe we can break this method down to avoid the issue? Or add a short comment about what the error is and why its being suppressed. 

Like:
```suggestion
    # Suppress McCabe complexity check MC0001 temporarily to avoid refactoring right now.
    def save(self, **kwargs):  # pylint: disable=arguments-differ, too-many-branches, useless-suppression; # noqa: MC0001
```

Reply:
@xitij2000 Thanks for this.